src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+function getAverageAge(sex) {
+  return sex
+    .map(person => person.died - person.born)
```

Comment:
Instead of using single-letter variable names like 'a' and 'b', use more descriptive names like 'sum' and 'ageDifference' in the reducer function.